runtime.md

```diff
@@ -29,6 +29,7 @@ The state of a container includes the following properties:
     This is provided so that consumers can find the container's configuration and root filesystem on the host.
 * **`annotations`** (map, OPTIONAL) contains the list of annotations associated with the container.
     If no annotations were provided then this property MAY either be absent or an empty map.
+* **`seccomp`** (map, OPTIONAL) contains additional TODO.
```

Comment:
```suggestion
* **`seccompFd`** (int, OPTIONAL) contains the file descriptor of the seccomp filter associated to the container when it has any `SCMP_ACT_NOTIFY` action. 
```

Perhaps we should also say it's only relevant when calling the sendSeccompFd hook?